wgpu-native/src/command/compute.rs

```diff
@@ -38,14 +42,14 @@ impl<B: hal::Backend> ComputePass<B> {
 
 #[no_mangle]
 pub extern "C" fn wgpu_compute_pass_end_pass(pass_id: ComputePassId) -> CommandBufferId {
-    let mut command_buffer_guard = HUB.command_buffers.write();
```

Comment:
I think the original variable name is easier to digest.

Reply:
Noted. I just made it consistent with the render pass code, will re-evaluate (and you are welcome to make a PR that changes names to be cleaner!)